scripts/test_crud.sh

```diff
@@ -5,36 +5,49 @@ NEUTRON_DIR=${NEUTRON_DIR:-../neutron}
 HOME=${NEUTRON_DIR}/data/test-1/
 HOME2=${NEUTRON_DIR}/data/test-2/
 KEY=demowallet1
+FAUCET=demowallet3
 ADMIN=neutron1m9l358xunhhwds0568za49mzhvuxx9ux8xafx2
-RLY2=neutron17dtl0mjt3t77kpuhg2edqzjpszulwhgzcdvagh
+RLY2=cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs
+
 BIN=neutrond
+GAIAD_BIN=gaiad
+
+TEST_WALLET=test_wallet
```

Comment:
didn't get this move. it was `KEY=demowallet1` from whose behalf all txs were executed, but you introduced `TEST_WALLET=test_wallet` (which also required initialisation) and used it as main actor instead. 1) what's the reason? 2) KEY became unused

Reply:
demowallet1 is used to run query relayer as well, in this case we will have two process that are using same address to produce transaction and it leads us to a lot of `sequence` problems.

Reply:
Also dedicated address per script run helps us to have more or less clean environment between script executions.

Reply:
makes sense, thanks

Reply:
> demowallet1 is used to run query relayer as well, in this case we will have two process that are using same address to produce transaction and it leads us to a lot of `sequence` problems.

maybe we should finally allocate an account for relayer? I am sick tired of this collision. I have proposed to do that N months ago but we decided to postpone it.

Reply:
and I will do it if we approve adding one more account :^)

Reply:
@foxpy I think we can achieve this in other PR, I will prepare task for Jira in the meantime

Reply:
Added https://p2pvalidator.atlassian.net/browse/NTRN-297